Source/CarthageKit/Xcode.swift

```diff
@@ -276,7 +276,8 @@ public func schemesInProjects(_ projects: [(ProjectLocator, [Scheme])]) -> Signa
 			switch project {
 			case .projectFile where !schemes.isEmpty:
 				return true
-
+            case .workspace where !schemes.isEmpty:
```

Comment:
The indentation is a little off here -- Carthage uses tabs rather than spaces so I think that is the issue.